packages/manager/modules/nasha/src/dashboard/partition/accesses/delete/delete.routing.js

```diff
@@ -0,0 +1,17 @@
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state('nasha.dashboard.partition.accesses.delete', {
+    url: '/:ip/delete',
```

Comment:
Makes no sense to have a `/:ip` state ?

Reply:
Can you please explain why ?

It simply mirrors the api :
DELETE /dedicated/nasha/{serviceName}/partition/{partitionName}/access/{ip}
If `/:serviceName` or `/:partitionName` make sense, why this doesnt ?

Also we have to keep track of the ip so the page can be refreshed without loosing the information. I could have done something like `/delete?ip` but what's the difference ?

Reply:
Actually it is not `/:ip` but `/:ip/delete`